instrumentation/instasarama/async_producer_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func TestAsyncProducer_Input(t *testing.T) {
-	headerFormats := []string{"binary", "string", "both"}
```

Comment:
`nitpicking` `opinionated` : `text` or `plaintext` format would have been better than `string` format. Just to back up my unnecessary argument, [binary strings](https://www.ibm.com/docs/en/db2/11.5?topic=list-binary-strings) are a thing in the IBM universe. ;)